guides/getting_started_command_line.rst

```diff
@@ -21,6 +21,8 @@ Installing ESPHome is very easy. All you need to do is have `Python
 
     pip install esphome
 
+Don't forget to add the esphome binary to PATH to ensure proper execution.
```

Comment:
I doubt the average user will understand what this is supposed to mean. Maybe it would fit better in the FAQ entry about windows. And definitely add a short code block showing what output is seen when it's not in the PATH (command not found or whatever it is on windows) - that makes the issue searchable.

Additionally, an example of where the binary usually is would be useful (guessing something like user appdata somewhere?)